src/WebJobs.Script/Content/Script/res.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+module.exports = () => {
+    var res = {};
```

Comment:
Pretty standard practice in express: http://expressjs.com/en/api.html#res, but not opposed to changing to `response`